### PR TITLE
introduce serial and document for holiday module

### PR DESCRIPTION
**Pull request by defrance**

### CODE REVIEW

**Comment by dolibarr95** [comment]
`holiday_madona`
loool
 :-)

**Comment by defrance** [comment]
for the pdf file (still in dev on my side), i have "celebrate" and "everybody" for the odt

**Comment by dolibarr95** [comment]
I just love it !
lol :-)

File: htdocs/admin/holiday.php [hunk 1]
```diff
@@ -0,0 +1,521 @@
...
+$dirmodels=array_merge(array('/'),(array) $conf->modules_parts['models']);
+
+llxHeader();
+
+$form=new Form($db);
+
+$linkback='<a href="'.DOL_URL_ROOT.'/admin/modules.php">'.$langs->trans("BackToModuleList").'</a>';
+print load_fiche_titre($langs->trans("HolidaySetup"),$linkback,'title_setup');
+
+$head=holiday_admin_prepare_head();
```
Comment (eldy) [inline htdocs/admin/holiday.php:195]: This function  holiday_admin_prepare_head  is not found !

**Comment by eldy** [comment]
@defrance 
I get an error on holiday_admin_prepare_head function not found in develop branch

**Comment by ariaconseil** [comment]
> for the pdf file (still in dev on my side), i have "celebrate" and "everybody" for the odt

Hi @defrance , is there any update for the module for the PDF file please ?

**Pull request merged**